assets/js/blocks/cart-checkout/checkout/edit.js

```diff
@@ -58,9 +57,9 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
 	// Checkout signup is feature gated to WooCommerce 4.7 and newer;
 	// uses updated my-account/lost-password screen from 4.7+ for
 	// setting initial password.
-	// Also currently gated to dev builds only.
-	const showCreateAccountOption =
-		isExperimentalBuild() && compareWithWooVersion( '4.7.0', '<=' );
+	// Also implicitly gated to feature plugin, because Checkout
+	// block is gated to plugin
```

Comment:
I wanted a full stop on this comment, but eslint action told me to remove it https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3331/commits/cc4f7ed0ef9bdd501fd1c7c99b422b326c508fe7!

Reply:
I wonder if the lint rule got confused by the `Checkout` sentence case at the end of the line. Might be a bug in the rule.